src/services/refactors/useDefaultImport.ts

```diff
@@ -0,0 +1,96 @@
+/* @internal */
+namespace ts.refactor.installTypesForPackage {
+    const actionName = "Convert to default import";
+
+    const useDefaultImport: Refactor = {
+        name: actionName,
+        description: actionName,
+        getEditsForAction,
+        getAvailableActions,
+    };
+
+    registerRefactor(useDefaultImport);
+
+    function getAvailableActions(context: RefactorContext): ApplicableRefactorInfo[] | undefined {
+        const { file, startPosition, program } = context;
+
+        if (!program.getCompilerOptions().allowSyntheticDefaultImports) {

```

Comment:
@weswigham we will need to add your new commandline argument to enable the interop behavior here as well.

Reply:
If you go look at the PR, I have a similar but more precise quickfix which triggers on a diagnostic added when a namespace object is misused.

Reply:
I think we need both. The quick fix for the error, the refactoring for style change even without an error.